neo.UnitTests/Network/RPC/UT_TransactionManager.cs

```diff
@@ -158,7 +158,7 @@ public void TestSignMulti()
                 .Sign();
 
             var store = TestBlockchain.GetStore();
-            var snapshot = store.GetSnapshot();
+            var snapshot = store.GetSnapshot().Clone();
```

Comment:
Why this change?

Reply:
Is for parallel execution, without Clone every test alter the storage of the others, but i want to remove the parallelization changes.

Reply:
These changes (Clone) should come with the isolation of the unit test, i will reduce the changes only for the problem with travis, a small optimizations